index.js

```diff
@@ -11,11 +11,13 @@ module.exports = {
       ignoreComments: true,
       ignoreStrings: true,
     }],
+    'import/extensions': 'off',
     'import/no-commonjs': 'error',
     'eol-last': ['error', 'always'],
     'no-unused-expressions': ['error', { allowTernary: true }],
     'arrow-body-style': 'off',
     'func-names': 'off',
     'react/jsx-boolean-value': 'off',
+    'jsx-a11y/no-static-element-interactions': 'off',
```

Comment:
Looks like it is fixed. Supports ^5.1.1 @ https://github.com/airbnb/javascript/blob/fc7fae620f625701a5614fb9cf9ec1e9c49bcc70/packages/eslint-config-airbnb/package.json